docs/docs.md

```diff
@@ -168,11 +171,22 @@ The [`config/log.rego`](../config/log.rego) file ensures that sensitive informat
 
 ## Component 3: the integrator authentication endpoint
 
-The nodejs server provides an [`/authenticated`](../example-app/index.js#L56) endpoint which can be used to check if the caller is logged in.
+The nodejs server provides an [`/jsonwebtoken`](../example-app/index.js#L57) endpoint which can be used to generate a signed JSON Web Token authorizing the request.
 
-This is called by the application itself and by the Open Policy Agent in the `webapp.rego` script.
+This is called by the example-app and then included in the `authorization` header
+with the request to the envoy proxy.
 
-In this example application, this authentication component has been simplified to illustrate the allow and reject states.
+The envoy proxy then [validates this JWT](../config/webapp.rego#L25) as part of
+the request processing to ensure that the caller is authorized before forwarding
+the request to OpenAI.
+
+This could also be implemented by passing the session cookie to the proxy and
+having the proxy call the `/authenticated` endpoint.
```

Comment:
```suggestion
The envoy proxy then [validates this JWT](../config/webapp.rego#L25) as part of
the request processing. This ensures the caller is authorized before forwarding
the request to OpenAI.

Alternatively, implement this by passing the session cookie to the proxy and
having the proxy call the `/authenticated` endpoint.
```
1. Divided independent clauses into sentences.
2. Switched from adverbial preposition, _to_, to referent pronoun, _This_, in the process.
3. Edited following sentence to avoid starting two sentences with a referent pronoun.